cmake/openvdb-extras.cmake.in

```diff
@@ -1,4 +1,3 @@
 # This overrides the dependency tracker with the OpenVDB library file.
-file(GLOB OpenVDBLibraries @CATKIN_DEVEL_PREFIX@/lib/libopenvdb*)
-set( @PROJECT_NAME@_LIBRARIES ${OpenVDBLibraries} tbb Half)
-
+set(OpenVDBLibraries "${CATKIN_DEVEL_PREFIX}/lib/libopenvdb.so")
```

Comment:
this won't work on osx where the libraries have a different extension. Please see the thirdparty submodules packages for examples how to set the extension using cmake.